.travis.yml

```diff
@@ -6,6 +6,7 @@ rvm:
   - 2.2.6
   - 2.3.3
   - 2.4.0
+  - 2.5.0
```

Comment:
Should we test against all of the latest patch versions while we're here?

- 2.2.9
- 2.3.6
- 2.4.3
- 2.5.0

Reply:
Should we go ahead and remove the older patch versions? Does it do us much good to test against 2.2.6 if we're testing against 2.2.9?

Reply:
Yes, replace the ones we have in there now.